lib/galaxy/tools/imp_exp/__init__.py

```diff
@@ -74,6 +74,11 @@ def get_tag_str( tag, value ):
                 #
                 # Create history.
                 #
+                if not os.path.exists(os.path.join( archive_dir, 'history_attrs.txt')):
```

Comment:
Sorry for the piecemeal review, but could you drop the whitespace before `archive_dir`, move the added code before the existing comment and add a comment saying something like `'# Galaxy pre 17.01 stored exported histories in a subdirectory` ?